recipes/natural_language_processing/chatbot/Makefile

```diff
@@ -1,19 +1,19 @@
 APP=chatbot
-MODELIMAGE=quay.io/ai-lab/mistral-7b-instruct:latest
-APPIMAGE=quay.io/ai-lab/${APP}:latest
-SERVERIMAGE=quay.io/ai-lab/llamacpp-python:latest
-SSHPUBKEY := $(shell cat ${HOME}/.ssh/id_rsa.pub;)
+MODELIMAGE ?= quay.io/ai-lab/mistral-7b-instruct:latest
+APPIMAGE ?= quay.io/ai-lab/${APP}:latest
+SERVERIMAGE ?= quay.io/ai-lab/llamacpp-python:latest
+SSHPUBKEY ?= $(shell cat ${HOME}/.ssh/id_rsa.pub;)
 .PHONY: build
 build:
 	podman build -f builds/Containerfile -t ghcr.io/ai-lab-recipes/${APP} .
 
 .PHONY: bootc
 bootc:
-	podman build --cap-add SYS_ADMIN --build-arg "SSHPUBKEY=$(SSHPUBKEY)" -f bootc/Containerfile -t ghcr.io/ai-lab-recipes/${APP}-bootc .
+	sudo podman build --cap-add SYS_ADMIN --build-arg "SSHPUBKEY=$(SSHPUBKEY)" -f bootc/Containerfile -t ghcr.io/ai-lab-recipes/${APP}-bootc .
```

Comment:
It's tricky though because then this `Makefile` won't do the right thing on MacOS (and I think we want that right?)

So this goes back to using podman-machine even on Linux right?

Reply:
however, on my Mac with podman v5 - I need to add `--network=host` in order to build this.

Reply:
and also, on Mac, I cannot run nested podman cmds during podman build

Reply:
I removed the sudo.

Reply:
What it really means is requiring users to use rootful containers or figuring out why this is not working on rootless  mode.